DB Setup/createDB.sql

```diff
@@ -32,7 +33,8 @@ create table thsSurvey (
 	CompletionTime Time,
 	THS_SectionA int,
 	THS_SectionB int, 
-	THS_SectionC text,
+	THS_SectionC int,
+        THS_SectionC_Exmaple text, 
```

Comment:
Some lines seem to have extra whitespace for some reason

Reply:
I tried fixing them but it wont go away and im not sure why